### ISSUE TITLE
Better better messages needed

### ISSUE THREAD

**Issue by kalpaitch** [warmshowers/Warmshowers.org#595]
Better Messages is a very simple module that provides "Popup-like" Drupal messages.

The main reason for this is to make sure users can't miss the important site warnings we give them.

However, popups are not good for the mobile experience and we need to move away from any popups on the site to support the growing mobile usage (approximately 34% from Jan 1st to June 26th 2015).

Suggesting incorporating this into the site-wide redesign, if not sooner.

**Comment by rfay** [comment]
I'm good with doing something different. The traditional Drupal thing is completely inadequate, as nobody sees these at all, and so they get confused, especially when editing profile.  Better messages at least puts it in their face (but has its own problems)

**Comment by kalpaitch** [comment]
Yes, I agree. This will take quite a bit of thought re: design and prominence.
